### PR TITLE
Feature/baseline metaanalysis unify

### PR DESCRIPTION
**Pull request by suhaibMo**
In this PR,
- We are unifying RUV normalisation script to make generic one to run `tissues`(GTEX) or `cell_types` (BLUEPRINT)
- More Env Var declaration for parameters and parsing experiment accessions via Jenkins
- When combing matrices we were under the assumption that all the baseline studies are mapped to a same E!Genome assembly which results similar number of genes. However in this release, we found discrepancies as one study was aligned with latest version and other consortium studies with older version resulting in inconsistent number of genes.

: Testing 
http://193.62.52.166:30752/jenkins/view/Bulk%20data%20exports/job/AA_OpenTargets_Baseline_Export/9/console

### CODE REVIEW

File: bin/RUV_normalisation.R [hunk 1]
```diff
@@ -38,35 +47,42 @@ for (i in names(t)) {
   exp <- k$rnaseq
   eCounts <- assays(exp)$counts
   samples<-colnames(eCounts)
-  average.counts<-technical_replicate_average_gtex(exp,expAcc)
-  all <- cbind(all,average.counts)
+  if ( study_type == "tissues" ){
+    average.counts<-technical_replicate_average_gtex(exp,expAcc)
```
Comment (pcm32) [inline bin/RUV_normalisation.R]: Is this still only applicable to gtex only? I thought that the idea was that this was more generally applicable, but maybe I got that wrong.

Comment (pcm32) [reply bin/RUV_normalisation.R]: Same below with blueprints

File: bin/export_ot_baseline_metaanalysis.sh [hunk 2]
```diff
@@ -4,12 +4,12 @@ scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 [ ! -z ${ATLASPROD_PATH+x} ] || ( echo "Env var ATLASPROD_PATH path to the directory needs to be defined." && exit 1 )
 [ ! -z ${BASELINE_META_DESTINATION+x} ] || ( echo "Env var BASELINE_META_DESTINATION path to the baseline baseline meta analysis directory needs to be defined." && exit 1 )
-[ ! -z ${BLUEPRINT_STUDIES+x} ] || ( echo "Env var BLUEPRINT_STUDIES comma separated blueprint baseline studies needs to be defined." && exit 1 )
-[ ! -z ${GTEX_STUDIES+x} ] || ( echo "Env var GTEX_STUDIES comma separated gtex baseline studies needs to be defined." && exit 1 )
+[ ! -z ${CELLTYPE_STUDIES+x} ] || ( echo "Env var BLUEPRINT_STUDIES comma separated blueprint baseline studies needs to be defined." && exit 1 )
```
Comment (pcm32) [inline bin/export_ot_baseline_metaanalysis.sh]: Please make this and following echo consistent (shouldn't be GTEX or BLUEPRINTS any more).

**Review by pcm32 — APPROVED** [review]
Excellent reduction of redundancy. A few minor things to improve this, please merge once fixed.

File: bin/RUV_normalisation.R (hunk 1 above)
Comment (suhaibMo) [reply bin/RUV_normalisation.R]: Good catch 👍.  Updated now with renaming the function `_gtex` to `_tissues`. Same for blueprint replacing with `_bp` to `_cell_types`

File: bin/export_ot_baseline_metaanalysis.sh (hunk 2 above)
Comment (suhaibMo) [reply bin/export_ot_baseline_metaanalysis.sh]: Done

**Comment by suhaibMo** [comment]
Thanks @pcm32

**Pull request merged**